apps/web/frontend/components/weekSelector.tsx

```diff
@@ -47,6 +51,32 @@ export const WeekSelector = ({ value, onChange }: WeekSelectorProps) => {
           <FaChevronRight />
         </button>
       </div>
+      {myProjectsMembersData.length > 0 && (
+        <Listbox
+          value={myProjectsMembersData.find((user) => user.id === selectedUserId) ?? myProjectsMembersData[0]}
+          getLabel={(user) => (
+            <div className="flex items-center gap-2">
+              {user?.image ? (
+                <Image
+                  src={user.image}
+                  alt={user.name ?? 'User avatar'}
+                  width={26}
+                  height={26}
```

Comment:
```suggestion
                  width={24}
                  height={24}
```
placeholder seems to be 24x24, so let's keep it consistent

Reply:
Done